cvat/apps/engine/models.py

```diff
@@ -256,29 +256,29 @@ def get_original_cache_dirname(self):
         return os.path.join(self.get_data_dirname(), "original")
 
     @staticmethod
-    def _get_chunk_name(chunk_number, chunk_type):
+    def _get_chunk_name(segment_id: int, chunk_number: int, chunk_type: DataChoice | str) -> str:
         if chunk_type == DataChoice.VIDEO:
             ext = 'mp4'
         elif chunk_type == DataChoice.IMAGESET:
             ext = 'zip'
         else:
             ext = 'list'
 
-        return '{}.{}'.format(chunk_number, ext)
+        return 'segment_{}-{}.{}'.format(segment_id, chunk_number, ext)
 
-    def _get_compressed_chunk_name(self, chunk_number):
-        return self._get_chunk_name(chunk_number, self.compressed_chunk_type)
+    def _get_compressed_chunk_name(self, segment_id: int, chunk_number: int) -> str:
+        return self._get_chunk_name(segment_id, chunk_number, self.compressed_chunk_type)
 
-    def _get_original_chunk_name(self, chunk_number):
-        return self._get_chunk_name(chunk_number, self.original_chunk_type)
+    def _get_original_chunk_name(self, segment_id: int, chunk_number: int) -> str:
+        return self._get_chunk_name(segment_id, chunk_number, self.original_chunk_type)
 
-    def get_original_chunk_path(self, chunk_number):
+    def get_original_segment_chunk_path(self, chunk_number: int, segment: int) -> str:
```

Comment:
segment_id to be consistent with the code above?

Reply:
Fixed